dist/index.html

```diff
@@ -0,0 +1,67 @@
+<!doctype html>
```

Comment:
dist složka nepatří do repozitáře :D 

rule of thumb: jakmile je něco výstupem nějakýho build kroku (kterym je i instalace závislostí), tak to nepatří do repa